src/app/guides/page.tsx

```diff
@@ -195,9 +195,9 @@ const StyleGuide: FC = () => {
           <span>Import the custom button component</span>
           <div className="rounded-lg bg-zinc-950 p-4">
             <span className="block font-mono text-sm text-white">
-              import <span className="text-blue-400">CustomButton</span> from{" "}
+              import <span className="text-blue-400">CustomButton</span> from
               <span className="text-yellow-400">
-                &quot;~/components/common/Button/button&quot;
+                `~/components/common/Button/button`
```

Comment:
why did you change this line?

Reply:
I don't know how that happened but I've changed it back.
Thank you.